src/components/Banner/MucBanner.vue

```diff
@@ -6,12 +6,22 @@ import { computed } from "vue";
 type bannerType = "info" | "warning" | "emergency";
 
 const props = defineProps({
+  /**
+   * Changes the style of the banner. Available types are `info`, `warning` and `emergency`.
+   */
   type: {
     type: String as PropType<bannerType>,
     default: "info",
   },
 });
```

Comment:
```suggestion
const props = withDefaults(
  defineProps<{
    /**
     * Changes the style of the banner. Available types are `info`, `warning` and `emergency`.
     */
    type?: bannerType;
  }>(),
  {
    type: "info",
  }
);
```
I didn't change this in the prettier introduction, but we should use the simpler `pure types` variant when defining props and slots (beneath this, `defineSlots`already uses this variant like any other component).